pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -86,7 +86,7 @@ const CmScanResultAnnotation = "compliance.openshift.io/scan-result"
 const CmScanResultErrMsg = "compliance.openshift.io/scan-error-msg"
 
 const (
-	// ResultNot available represents the compliance scan not having finished yet
+	// ResultNotAvailable available represents the compliance scan not having finished yet
```

Comment:
should be `// ResultNotAvailable represents the compliance scan not having finished yet`